v2/samples/dbformysql/v1api/v1api20210501_flexibleserver.yaml

```
@@ -17,3 +17,11 @@ spec:
     key: password
   storage:
     storageSizeGB: 128
+  # Use UMI as identity - this is required for setting up an AAD Admin but can be omitted otherwise
```

Probably best to define new acronyms in samples, as users may not know them.

```suggestion
  # Use User Managed Identity (UMI) - this is required for setting up an AAD Admin but can be omitted otherwise
```

---

Agreed. Fixed.